### PR TITLE
a space odyssey (addressing multiple issues for 1.9.1)

### PR DESCRIPTION
**Pull request by gregcaporaso**
image

### CODE REVIEW

**Comment by jairideout** [comment]
Ready for merge when tests pass!

File: qiime/util.py [hunk 1]
```diff
@@ -555,6 +559,11 @@ def write_biom_table(biom_table, biom_table_fp, compress=True,
         The Table.type value to set for the table before it is written. Note
         that this is a controlled vocabulary documented on biom-format.org.
     """
+    if biom_table.is_empty():
```
Comment (ElDeveloper) [inline qiime/util.py]: Can you add this exception to the Raises section of the docstring?

**Comment by ElDeveloper** [comment]
:+1:, thanks Dave. :rocket:

**Comment by qiime-r2d2** [comment]
Build results will soon be (or already are) available at: http://ci.qiime.org/job/qiime-github-pr/1633/

**Pull request merged**